app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -353,10 +353,10 @@ def alert_profile_sync_provider(old_alerts = nil, new_alerts = nil)
       if old_alerts.nil? && new_alerts.nil?
         operation = :update_assignments
         old_alerts = new_alerts = @alert_profile.miq_alerts.collect(&:id)
-        assigned = @alert_profile.get_assigned_tos
       else
         operation = :update_alerts
       end
+      assigned = @alert_profile.get_assigned_tos
```

Comment:
https://github.com/lucasponce/manageiq/blob/92c9ad218f1d1dc85f5082e3212e1411b9436dc0/app/controllers/miq_policy_controller/alert_profiles.rb#L352
This is only related for Middleware use cases.